### PR TITLE
Resize according to short edge

### PR DESCRIPTION
**Pull request by yl-1993**

### CODE REVIEW

**Comment by xiaojieli0903** [comment]
The unit test is failed. Please fix it. @yl-1993

**Comment by xiaojieli0903** [comment]
Please update the docstring for `size`. @yl-1993

File: mmcls/datasets/pipelines/transforms.py [hunk 1]
```diff
@@ -328,6 +328,7 @@ class Resize(object):
 
     Args:
         size (int | tuple): Images scales for resizing (h, w).
+            When size is int, the short edge of an image is resized to `size`.
```
Comment (hellock) [inline mmcls/datasets/pipelines/transforms.py:331]: This behavior breaks the convention and may confuse users. I suggest using `(224, -1)` or `(224, inf)` to represent resizing by the short edge.

Comment (yl-1993) [reply mmcls/datasets/pipelines/transforms.py]: Thanks. We use the `(size, -1)` to represent resizing by the short side. When size is int, the default behavior is kept to resize an image to `(size, size)`.

**Pull request merged**